service/crawlingKeywordService.js

```diff
@@ -49,10 +48,9 @@ const getCrawlingKeyword = async (req, res) => {
       }
       if (hasKeywordOfIframe) {
         return res.status(200).json({
-          urlLink: req.params.url,
-          keyword: req.query.keyword,
+          url: req.params.url,
           hasKeyword: hasKeywordOfIframe,
-          urlText: iframeInnerText,
+          urlText: iframeInnerText.replace(/\n|\r|\t/g, ""),
```

Comment:
위 코드와 중복되는 부분인것 같습니다.  아무래도 iframeInnerText의 html body 본문이기 때문에 그 양이 많을 거라고 생각이 되는데, replace를 변수에 저장 없이 두 번 사용함에 따라 얻는 비효율이 있을 것 같습니다.
그래서 저는 처음에 한 번 replace할 때 변수에 저장해놓고 후에 그 변수를 가지고 사용하면 더욱 좋은 코드가 될 것 같다고 생각합니다.
제가 replace가 성능에 안좋다고 한 이유에 대한 [참조 링크](https://stackoverflow.com/questions/10398931/how-to-remove-text-from-a-string/65724578#65724578) 입니다.
```suggestion
          urlText: replaceInnerText,```

Reply:
네, 현재 iframe 페이지와 기본 페이지를 처리하는 부분이 분리되어 있어 어쩔 수 없이 replace를 두 번 사용하게 되었습니다. 또한 남겨주신 리뷰 덕분에 replace의 성능에 대해서도 생각해볼 수 있는 기회가 되었습니다. 좋은 의견 감사합니다!